### PR TITLE
WIP: 4.0.0 (monorepo)

### PR DESCRIPTION
**Pull request by zMotivat0r**

### CODE REVIEW

**Comment by ignatvilesov** [comment]
Hi @zMotivat0r 

I see this PR is for version 4.0.0. Could you please share any release date?

**Comment by zMotivat0r** [comment]
@ignatvilesov I wish that could be done easily 😞 Currently I'm on `@nestjsx/crud-typeorm` package, after that there will be work on release notes, migration notes, new documentation, etc. I think, generally, that might take a week plus/minus a couple of days

**Comment by ignatvilesov** [comment]
Thanks for the clarification @zMotivat0r 
We're really looking forward to using the new version as we took this library for our new project 😄

**Comment by zMotivat0r** [comment]
@ignatvilesov I totally understand your concerns, man. But since the lib usage is super easy (imo, it's a very little amount of code you need to write, right?) the migration process will not be very hard at all.

**Comment by ignatvilesov** [comment]
Right. Thanks!

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1 @@
+workspaces-experimental true
```
Comment (Bnaya) [inline .yarnrc]: This is no longer needed :)

**Pull request merged**